test/helper.rb

```diff
@@ -3,13 +3,13 @@
 require File.expand_path("../../test/dummy/config/environment.rb",  __FILE__)
 
 require 'minitest/autorun'
-
-FakeWeb.allow_net_connect = %r[^https?://127.0.0.1]
+require 'webmock/minitest'
 
 module MiniTest
   class Test
     def setup
-      FakeWeb.clean_registry
+      WebMock.reset!
+      WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
Fixed in https://github.com/data-axle/elastic_record/pull/46/commits/a67637c1ee3ed832294e8bacf9b83209678d1c49